extensions/community_extensions/skadnetwork.md

```diff
@@ -82,30 +82,30 @@ If a DSP has at least one SKAdNetworkItem in the publisher app’s `Info.plist`
   <tbody>
     <tr>
       <td>
-        <code>version</code>
+        <code>versions</code>
       </td>
       <td>
-        Version of skadnetwork supported. Always "2.0" or higher. Dependent on both the OS version and the SDK version. </br></br><strong>Note</strong>: With the release of SKAdNetwork 2.1, this field is deprecated in favor of the <code>BidRequest.imp.ext.skadn.versions</code> to support an array of version numbers.
+        Array of strings containing the supported skadnetwork versions. Always "2.0" or higher. Dependent on both the OS version and the SDK version.
```

Comment:
Not sure if we should keep the current phrasing "Always "2.0" or higher." since it's an array and not a specific value

Reply:
Generally, the thought here was that version 1.0 was not supported so 2.0 or beyond would be the expected values returned. Given that context, should we still remove it?

Reply:
in that case i guess we can keep it